lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -142,7 +135,7 @@ export default function FilePickerApp({ onSubmit }) {
           ...deepLinkingAPI.data,
           content,
           extra_params: {
-            groupSet: groupConfig.useGroupSet ? groupConfig.groupSet : null,
+            group_set: groupConfig.useGroupSet ? groupConfig.groupSet : null,
```

Comment:
This was a bug in the merged PR, the name the backend is expecting is `group_set`